packages/wonder-blocks-modal/src/components/modal-content.tsx

```diff
@@ -48,9 +38,9 @@ ModalContent.isComponentOf = (instance: any): boolean => {
  * TODO(WB-1655): Change this to use the theme instead (inside themedStylesFn).
  * e.g. `[theme.breakpoints.small]: {...}`
  */
-const small = "@media (max-width: 767px)";
+const small = "@media (max-width: 767px)" as any;
```

Comment:
I wonder if we can use the media query tokens for this! https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-tokens/src/tokens/media-queries.ts

Reply:
I initially thought about this, but ended up not changing it as there's no current WB query equivalent. This is something that we surely need to address once we migrate our overall MQ strategy to use these official WB tokens.